plugins/git-release-manager/src/features/Stats/Row/Row.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import { DateTime } from 'luxon';
+import {
+  Collapse,
+  IconButton,
+  Link,
+  makeStyles,
+  TableCell,
+  TableRow,
+} from '@material-ui/core';
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp';
+
+import { ReleaseStats } from '../contexts/ReleaseStatsContext';
+import { RowCollapsed } from './RowCollapsed/RowCollapsed';
+
+const useRowStyles = makeStyles({
+  root: {
+    '& > *': {
+      borderBottom: 'unset',
+    },
+  },
+});
+
+interface RowProps {
+  baseVersion: string;
+  releaseStat: ReleaseStats['releases']['0'];
+}
+
+export function Row({ baseVersion, releaseStat }: RowProps) {
+  const [open, setOpen] = useState(false);
+  const classes = useRowStyles();
+
+  return (
+    <>
+      <TableRow className={classes.root}>
+        <TableCell>
+          <IconButton
+            aria-label="expand row"
+            size="small"
+            onClick={() => setOpen(!open)}
+          >
+            {open ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+          </IconButton>
+        </TableCell>
+
+        <TableCell component="th" scope="row">
+          <Link href={releaseStat.htmlUrl} target="_blank">
+            {baseVersion}
+            {releaseStat.versions.length === 0 ? ' (prerelease)' : ''}
+          </Link>
+        </TableCell>
+
+        <TableCell>
+          {releaseStat.createdAt
+            ? DateTime.fromISO(releaseStat.createdAt)
+                .setLocale('sv-SE')
```

Comment:
does this make any difference? :)

Reply:
Oh, this is a good point. The timestamps are in UTC, but this locale should probably take the user into account.

Is the user's locale available somewhere or would I have to attempt to retrieve it from the browser?

Reply:
Correct me if I'm wrong, but the locale is not the same thing as the time zone. The locale would control the _format_ of the output if you stringified with something other than the strict year-month-day format specifiers that you have.

I would say, pick one (browser-local or UTC) that you think makes the most sense for users to see, for now. Probably keep UTC. And then we should add a component to the core offering, whose purpose it is to show timestamps like this. And then the user can select themselves (globally) what they prefer, even including relative specifiers ("3 minutes ago") etc

Reply:
> Probably keep UTC

Will probably go with this option

> And then we should add a component to the core offering

Niiice